gitlabform/processors/project/tags_processor.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, gitlab: GitLab, strict: bool):
         self.strict = strict
 
     def _process_configuration(self, project_and_group: str, configuration: dict):
-        project = self.gl.projects.get(id=project_and_group, lazy=True)
+        project = self.gl.get_project_by_name(name=project_and_group, lazy=True)
```

Comment:
I wonder why it needed `lazy=True` while the other processors above didn't need to do that. Why is this special 🤔

Reply:
yeah, that makes sense.